cppcache/src/Log.cpp

```diff
@@ -45,9 +45,21 @@ const int LOG_SCRATCH_BUFFER_SIZE = 16 * __1K__;
 static std::recursive_mutex g_logMutex;
 static std::shared_ptr<spdlog::logger> currentLogger;
 static LogLevel currentLevel = LogLevel::None;
+static std::string logFilePath;
+static int32_t adjustedFileSizeLimit;
+static int32_t maxFiles;
 
 const std::shared_ptr<spdlog::logger>& getCurrentLogger() {
-  return currentLogger;
+  if (logFilePath.empty()) {
```

Comment:
In the Java bits we landed on `-`, consistent with lots of utilities, as the indicator to use `stdout` logging.

Reply:
Modified the code to accept this as an explicit request for `stdout` logging.